scripts/after_platform_add.js

```diff
@@ -0,0 +1,25 @@
+var async = require('async');
+var spawn = require('./spawn');
+
+var addPlugin = function(plugin, callback) {
+    spawn('cordova', ['plugin', 'add', plugin], callback);
+};
+
+module.exports = function(context) {
+    var Q = context.requireCordovaModule('q');
```

Comment:
I swears it: https://github.com/apache/cordova-lib/blob/master/cordova-lib/templates/hooks-README.md#javascript